### ISSUE TITLE
Connection lost & regain & lost again & regain...

### ISSUE THREAD

**Issue by marcatsmci** [wireapp/wire-desktop#4086]
Wire version: Linux latest - 3.18.2925
Wire for web version: 2020-07-24-13-46
Operating system: Linux Mint 20 Cinnamon
Which antivirus software do you have installed: na

What steps will reproduce the problem?

1. Launch Wire Desktop and using it

What is the expected result?

- Stable connection

What is the actual result?

- For few days, the client on Linux Mint 20 Cinnamon is very boring, loosing connection, try to reconnect, reconnects and so on, every 2 to 20 seconds...
- Very annoying because the content of the window goes down and up each time the client looses the connection and regains it...
- I have a perfect & fast internet connection
- in Firefox I haven't seen the problem

![wire-connection](https://user-images.githubusercontent.com/47881838/88903304-c35d1f80-d253-11ea-81a2-1fb370b10202.png)
![internet](https://user-images.githubusercontent.com/47881838/88903610-42525800-d254-11ea-9d9f-0b68400b63f8.png)

Please provide any additional information below. Attach a screenshot if possible.

**Comment by mythsunwind** [comment]
This sounds like the websocket connection to our backend server is either interrupted or denied. Do you use any firewall that blocks outgoing connections?

**Comment by marcatsmci** [comment]
> This sounds like the websocket connection to our backend server is either interrupted or denied. Do you use any firewall that blocks outgoing connections?

No changes for weeks...

**Comment by mythsunwind** [comment]
> Any possibility to record some log ?

Yes, open the app with `wire-desktop --enable-logging`. Then after the issue happened use the menu **Help | Download Debug Logs** to get a zip with logs.

**Comment by mj162** [comment]
Good morning, I have the same issue too.  Wire 3.21.2936, the Wire AppImage on Linux launches, retrieves my mugshot and settles.  Then, 5-10 seconds later it begins an endless cycle of displaying a blue banner at the top of the window having the words: "TRYING TO CONNECT.  WIRE MAY NOT BE ABLE TO DELIVER MESSAGES." accompanied by a spinning circle for 2-3 seconds.  Then the blue banner message changes to: "NO INTERNET.  YOU WON'T BE ABLE TO SEND OR RECEIVE MESSAGES". Then it simply cycles indefinitely with the following repeated interminably in the logs:

`[2020-11-30 07:15:05] [@wireapp/desktop/preload-webview.js] Received amplify event "wire.webapp.team.info": "{"accentID":1,"name":"name.removed","picture":"data:application/octet-stream;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAEBAQEBAQEBAQEBAQEBAQEBAQEBAQE...","teamRole":"z.team.TeamRole.ROLE.NONE","userID":"74cdf395-552a-4624-9f7f-009066b78c6c","availability":0}", forwarding event ...`
`[2020-11-30 07:15:05] [@wireapp/desktop/preload-webview.js] Received amplify event "wire.webapp.lifecycle.unread_count" (count: "0"), forwarding event ...`
`[2020-11-30 07:15:14] [@wireapp/desktop/preload-webview.js] Received amplify event "wire.webapp.lifecycle.unread_count" (count: "0"), forwarding event ...`
`[2020-11-30 07:15:25] [@wireapp/desktop/preload-webview.js] Received amplify event "wire.webapp.lifecycle.unread_count" (count: "0"), forwarding event ...`
`[2020-11-30 07:15:38] [@wireapp/desktop/preload-webview.js] Received amplify event "wire.webapp.lifecycle.unread_count" (count: "0"), forwarding event ...`
`[2020-11-30 07:15:52] [@wireapp/desktop/preload-webview.js] Received amplify event "wire.webapp.lifecycle.unread_count" (count: "0"), forwarding event ...`
`...`

Now, we **_have_** just changed ISP and our reverse (PTR) DNS zone is not being served currently.  This _does_ cause issues in browsing since every tom dick and harry site cross-site linked slows-up first trying to do a reverse check.  That's unfortunate but expected behaviour.  What I don't expect is for any app -- and so far Wire is the **_only_** app I've seen this on -- to fail.  Telegram, Jitsi, Swift, yada.., all work just fine even with impaired DNS.
